src/runtime/virtcontainers/hypervisor_s390x.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) IBM Corp. 2021
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const (
+	// This is valid in other architectures, but varcheck will complain
+	// when setting it in common code as it will be regarded unused
+	procKernelCmdline = "/proc/cmdline"
+
+	// Secure Execution
+	// https://www.kernel.org/doc/html/latest/virt/kvm/s390-pv.html
+	seCPUFacilityBit = 158
+	seCmdlineParam   = "prot_virt"
+)
+
+var seCmdlineValues = []string{
+	"1", "on", "oN", "On", "ON", "y", "Y",
```

Comment:
It might be worth just specifying the lower-case values here and lower-casing the param found before comparison. Also, if we're supporting multiple values like this, what about `true` and `yes`? :)

Reply:
> It might be worth just specifying the lower-case values here and lower-casing the param found before comparison.

Done.
> Also, if we're supporting multiple values like this, what about `true` and `yes`? :)

`true` doesn't work; added `yes` (actually, once `1`, `on`, or `y` are present, any string following that is accepted (e.g. `yeah`), but let's stick with this).